resources/qml/Root.qml

```diff
@@ -164,6 +172,18 @@ Page {
         target: TimelineManager
     }
 
+    Connections {
+        function onOpenReadReceiptsDialog(rr) {
+            var dialog = readReceiptsDialog.createObject(timelineRoot, {
+                "readReceipts": rr,
+                "room": Rooms.currentRoom
+            });
+            dialog.show();
+        }
+
+        target: Rooms.currentRoom
+    }
```

Comment:
Can you move this into the TimelineView and attach to the room instead of the Rooms.currentRoom? Otherwise you can only open read receipts in one room once we allow people to open multiple windows each showing a different room (tear out rooms basically).

Reply:
Passing in the currentRoom here also is a bit racy that the rr and room don't match up.

Reply:
Sure, I'll be glad to do that.